MoyParsers/Wujizun.com article.yaml

```diff
@@ -0,0 +1,42 @@
+info:
+  name: Wujizun.com article
+  description: Parses a single Wujizun.com article.
+  type: article
+  domain: wujizun.com
+  path: \d\d\d\d/\d\d/\d\d/[^/]*\d+/
+  testPages:
+    - "https://wujizun.com/2018/08/26/mysdv1c54/"

```

Comment:
Can't load this page (th site says "Oops! That page can’t be found.")

Reply:
Fixed with a rebase.